minecraft/src/main/java/org/quiltmc/loader/impl/game/minecraft/patch/TinyFDPatch.java

```diff
@@ -45,21 +45,14 @@ public final class TinyFDPatch extends GamePatch {
 	private static final String DIALOG_TITLE = "Select settings file (.json)";
 
 	@Override
-	public void process(QuiltLauncher launcher, GamePatchContext context) {
+	public void process(QuiltLauncher launcher, String namespace, GamePatchContext context) {
 		if (launcher.getEnvironmentType() != EnvType.CLIENT) {
 			// Fix should only be applied to clients.
 			return;
 		}
 
-		String className = MORE_OPTIONS_DIALOG_CLASS_NAME;
 
-		// Only remap the classname when needed to prevent loading the mappings when not required in prod.
-		if (!launcher.getMappingConfiguration().getTargetNamespace().equals("intermediary")
-				&& QuiltLoader.getMappingResolver().getNamespaces().contains("intermediary")) {
-			className = QuiltLoader.getMappingResolver().mapClassName("intermediary", MORE_OPTIONS_DIALOG_CLASS_NAME);
-		}
-
-		final ClassNode classNode = context.getClassNode(className);
+		final ClassNode classNode = context.getClassNode(MORE_OPTIONS_DIALOG_CLASS_NAME);
```

Comment:
Revert these changes. `QuiltLoader.getMappingResolver().getNamespaces()` returns a set, which will contain just  `[ "intermediary" ]` at runtime and `[ "named", "intermediary" ]` in a development environment. (In the future this could also contain `hashed` if/when we implement it in loader).

Reply:
Ah, I forgot that `hashed` mappings were planned. I've re-added the bit of code now, so it should be fine :+1: